lua/toggleterm/terminal.lua

```diff
@@ -452,6 +452,7 @@ end
 ---Spawn terminal background job in a buffer without a window
 function Terminal:spawn()
   if not self.bufnr or not api.nvim_buf_is_valid(self.bufnr) then self.bufnr = ui.create_buf() end
+  if not self.id then self.id = self.id or next_id() end
```

Comment:
is the `self.id or next_id()` necessary in this clause? could it just be
```suggestion
  if not self.id then self.id = next_id() end
```
seems the `not self.id` has already proven that the id is falsy so no need for the or

Reply:
True, haha not sure how I thought that was needed